--- run_alphafold.py.orig       2025-04-03 16:58:15.403776577 +0100
+++ run_alphafold.py    2025-04-03 16:29:41.283577936 +0100
@@ -714,9 +714,11 @@
     # Fail early on incompatible devices, but only if we're running inference.
     gpu_devices = jax.local_devices(backend='gpu')
     if gpu_devices:
-      compute_capability = float(
-          gpu_devices[_GPU_DEVICE.value].compute_capability
-      )
+      compute_capability_s = str(gpu_devices[_GPU_DEVICE.value].compute_capability)
+      if compute_capability_s.startswith("gfx"):
+        compute_capability = 642.0
+      else:
+        compute_capability = float(compute_capability_s)
       if compute_capability < 6.0:
         raise ValueError(
             'AlphaFold 3 requires at least GPU compute capability 6.0 (see'