--- preprocessing.py.orig       2025-03-12 21:57:47.000000000 +0000
+++ preprocessing.py    2025-03-12 21:57:55.000000000 +0000
@@ -28,7 +28,7 @@
     if out_a3m.exists() and out_atab.exists() and out_hhr.exists():
         return out_a3m, out_hhr, out_atab
 
-    search_command = f"./{command} {fasta_file} {out_dir} {num_cpus} {ram_gb} {search_base} {template_database}"
+    search_command = f"{command} {fasta_file} {out_dir} {num_cpus} {ram_gb} {search_base} {template_database}"
     print(search_command)
     _ = subprocess.run(search_command, shell=True)
     return out_a3m, out_hhr, out_atab